ui/src/message_popup/mobile_prototype/mobile_interface.jsx

```diff
@@ -44,29 +44,29 @@ export default React.createClass({
     const goodExamples = _.map(this.props.question.examples, (example) => { return {type: 'Good', text: example}; });
     const badExamples = _.map(this.props.question.nonExamples, (example) => { return {type: 'Bad', text: example}; });
     const allExamples = _.shuffle(_.clone(goodExamples).concat(badExamples)); 
-    const messages = this.getInitialMessages(this.props.question);
     return ({
-      messages,
+      messages: [],
```

Comment:
I moved away from adding setting the InitialMessages here in getInitialState so that they could be added using the staggering from addMessages in componentDidMount.